test/conformance/main_test.go

```diff
@@ -86,7 +86,7 @@ func addSourcesInitializers() {
 	)
 	sourcesTestRunner.AddComponentSetupClientOption(
 		testlib.PingSourceTypeMeta,
-		setupclientoptions.PingSourceV1A2ClientSetupOption(pingSrcName,
+		setupclientoptions.PingSourceV1B1ClientSetupOption(pingSrcName,
```

Comment:
Can you also change https://github.com/knative/eventing/blob/725f78e1ccbcc4fb63d38e7916759e5698dd4bd3/test/e2e-conformance-tests.sh#L36?

Reply:
Oh yes, I would have missed that. Great - thanks @aliok!